pkg/apis/extensions/v1beta1/types.go

```diff
@@ -213,6 +213,10 @@ type DeploymentSpec struct {
 	// Value of this key is hash of DeploymentSpec.PodTemplateSpec.
 	// No label is added if this is set to empty string.
 	UniqueLabelKey *string `json:"uniqueLabelKey,omitempty"`
+
+	// Paused indicates that the deployment will not be processed by the deployment
```

Comment:
What's our current resize policy? Resize the largest RC (up or down)? If so, that seems ok, even while paused.